From: Denis <sciamano@users.noreply.github.com>
Date: Sat, 26 Aug 2023 21:55:41 +0200
Subject: cmake fix

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 85f7ea8..4f22d49 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.17)
 
 set(APP_NAME "audio-loop")
 
